src/o365/spo/commands/page/page-get.ts

```diff
@@ -50,8 +50,8 @@ class SpoPageGetCommand extends SpoCommand {
     };
 
     request
-      .get<PageItem>(requestOptions)
-      .then((res: PageItem): void => {
+      .get<any>(requestOptions)
```

Comment:
The response is `any` by default, so there is no need to specify `any` as type on `get`.